File: 9.Revision_OOPS_Fundamental/index_revision_oops_fundamental.php
<?php
class User {
    public string $name;
    public int $age;

    public function __construct(string $name, int $age){
        $this->name = $name;
        $this->age = $age;
    }

    public function greet():void{
        echo "Hello, I am ". $this->name. PHP_EOL;
    }
    public function showAge():void{
        echo "My age is ". $this->age. PHP_EOL;
    }
}

$user = new User("John", 25);
$user->greet();
$user->showAge();
?>

File: 9.Revision_OOPS_Fundamental/index_revision_oops_fundamental.java
class User {
    String name;
    int age;

    User(String name, int age){
        this.name = name;
        this.age = age;
    }

    void greet(){
        System.out.println("Hello, I am "+this.name)
    }
    void showAge(){
        System.out.println("My age is "+this.age)
    }
}

User user = new User("John", 25);
user.greet()
user.showAge()

// ya -> main class ka use karke ise call kar sakte h

/* public class Main {
    public static void main(String[] args){
        User user = new User("John", 25);

        user.greet();
        user.showAge();
    }
} */